ads/meg.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {loadScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function meg(global, data) {
+  validateData(data, ['code']);
+  const code = data.code;
+  const lang = global.encodeURIComponent(global.navigator.language);
+  const ref = global.encodeURIComponent(global.referrer);
+  const params = ['lang=' + lang, 'ref=' + ref].join('&');
+  const url = 'https://apps.meg.com/embedjs/' + code + '?' + params;
+  loadScript(global, url, () => {
+    global.context.renderStart();
```

Comment:
I've updated the pull request. Now `renderStart` and `noContentAvailable` have been wrapped into `onSuccess` and `onError` callbacks. Additionally, I call `noContentAvailable` if we are unable to load the main javascript bundle. I've added optional `onError` callback to the `loadScript` function. Unfortunately, I don't know what is the best way to test it.